python/tvm/relay/__init__.py

```diff
@@ -19,51 +19,56 @@
 import os
 from sys import setrecursionlimit
 
-from . import call_graph
 from . import base
 from . import ty
 from . import expr
 from . import type_functor
 from . import expr_functor
 from . import adt
-from . import analysis
+from . import prelude
+from . import loops
+from . import scope_builder
+from . import parser
+
 from . import transform
+from . import analysis
+from .analysis import call_graph, feature, alpha_equal
```

Comment:
If analysis is a sub-namespace that we intend to ask user to use, consider not exposing these functions to the top-level, except for most common ones(e.g. alpha_equal)